containers/hydra/Dockerfile

```diff
@@ -3,7 +3,7 @@
 
 # Container for the hydra
 
-FROM nixos/nix
+ROM nixos/nix:2.17.0
```

Comment:
FROM

Reply:
Missed that while focusing on other things.

Reply:
ups, fixed and merging